app/views/world_locations/index.html.erb

```diff
@@ -8,6 +8,8 @@
         title: sanitize("Help and services around the&nbsp;world")
       } %>
 
+      <p class="govuk-body">Find out what is help is available from the UK government if you're abroad.</p>
```

Comment:
Can these strings go in the locales file? Looks like there are some others on this template that could do with shifting over there too if you've time.

Reply:
Good point I did think that and forgot!